test/testdrive/temporary.td

```diff
@@ -67,6 +67,19 @@ catalog item 'foo' already exists
 
 > CREATE OR REPLACE TEMPORARY MATERIALIZED VIEW foo AS SELECT * FROM v
 
+> CREATE TEMPORARY TABLE temp_t (a int, b text NOT NULL)
+
```

Comment:
the ```/dt``` command in ```psql``` does not list the temporary tables. Contrast with Postgres:

```
philips=> create temporary table t1 (f1 integer);
CREATE TABLE
philips=> \dt
         List of relations
  Schema   | Name | Type  |  Owner  
-----------+------+-------+---------
 pg_temp_3 | t1   | table | philips
(1 row)
```

Reply:
This issue is certainly related, but a bit removed from the code in this PR. I'm going to reopen our `\dt` issue and address it there: https://github.com/MaterializeInc/materialize/issues/4284